pyproject.toml

```diff
@@ -51,7 +51,7 @@ dependencies = [
     'typing-extensions>=4.6.1; python_version < "3.13"',
     'typing-extensions>=4.12.2; python_version >= "3.13"',
     'annotated-types>=0.6.0',
-    "pydantic-core==2.24.0",
+    "pydantic-core@ git+https://github.com/ollz272/pydantic-core.git@add-datetime-serializer",
```

Comment:
Looks good, we can leave this until we merge your `pydantic-core` PR just for testing purposes on this end.